contracts/tests/test_channel_create.py

```diff
@@ -115,6 +117,56 @@ def test_channel_223_create_bounty_limit(get_block, owner, get_accounts, uraiden
     assert channel_data[1] == channel_deposit_bugbounty_limit
 
 
+def test_create_token_fallback_uint_conversion(
+    contract_params,
+    owner,
+    get_accounts,
+    uraiden_instance,
+    token_instance):
+    token = token_instance
+    (sender, receiver) = get_accounts(2)
+
+    # Make sure you have a fixture with a supply > 2 ** 192 + 100
```

Comment:
Shouldn't you put an `assert` instead of a comment here?

Reply:
I was using an array of contract parameters to run the tests multiple times. But this is an edge case, so I will handle it only in this test with clear parameters. Refactor on the way.